### ISSUE TITLE
Upgrade Buttons extension

### ISSUE THREAD

**Issue by stla** [rstudio/DT#676]
The latest Buttons extension allows to set [attributes](https://datatables.net/reference/option/buttons.buttons.attr) to the buttons, such as an id or a title to get a tooltip. This is not available with the current version in `DT`. I have a use case where I need to select a button, I managed to select it but it would be more convenient with an id. Also setting a tooltip could be useful.

**Comment by yihui** [comment]
Upgrading DataTables and its extensions is not a trivial decision to make. Here is how I usually do it: https://github.com/rstudio/DT/blob/master/tools/update_DT.R

The DataTables version is defined in https://github.com/rstudio/DT/blob/master/R/package.R. What I'd have to do is to change the version number to the current 1.10.19, run `update_DT.R`, install **DT**, and do some tests. The last step is the trickiest since we don't have automated tests in this package... At least I need to make sure these examples work: https://github.com/rstudio/DT/tree/master/inst/examples Then rebuild the **DT** documentation site in the `gh-pages` branch and also make sure all tables still work well: https://github.com/rstudio/DT/blob/gh-pages/site.R If anyone can help me with the last step, I'll be happy to upgrade. Thanks!

**Comment by stla** [comment]
Ok. This is not important actually. I've found that we can set a title attribute with the `titleAttr` option, and we can set a class with `className`: 

```
            buttons = list(
              list(
                extend = "collection",
                text = 'Select all rows', 
                action = JS(restore), 
                titleAttr = "Restore the excluded rows"
                className = "mybutton"
              )
            )
```

Once we have set a class, we can conveniently select the button.

However it would be certainly nice to update anyway. For instance the latests `AutoFill` extension has new options, like `horizontal = FALSE` to prevent horizontal filling.

I have seen that the extensions do not provide `css` files but `scss` files. Are they already compiled if we download via the download builder ?
